app/src/getters/getClientConfig.js

```diff
@@ -0,0 +1,39 @@
+import { STREAM_CLIENT_DEFAULTS } from 'streamr-client'
+import formatConfigUrl from '$utils/formatConfigUrl'
+import formatRpc from '$utils/formatRpc'
+import formatTrackers from '$utils/formatTrackers'
+import config from '$config'
+
+export default function getClientConfig(mods = {}) {
+    const { client } = config
+
+    const { dataUnion, ensCacheChainAddress, ...DEFAULTS } = STREAM_CLIENT_DEFAULTS
```

Comment:
`ensCacheChainAddress` gets rejected by streamr-client's own config schema validation…

Reply:
@samt1803 have a look?

Reply:
Yes, the `ensCacheChainAddress` is in the ConfigBase, but according to the validation it shouldn't exist in the config.
At the moment it is not used anywhere, is was added for the case when we have a "reset cache" function, and that is scrapped for now, so `ensCacheChainAddress` can be removed completely. We can add it to config and validation once we need it.

Reply:
Thanks guys for chaiming in on this. I'll keep in mind to check on it again in a while and remove the patching I made here.